src/javascripts/components/tabs.js

```diff
@@ -33,45 +36,53 @@
       $(id).show().removeAttr('aria-hidden')
     },
 
-    // Accordion mode
-    clickAccordionItem: function (e) {
+    // Tabs mode
+    clickTabItem: function (e) {
       e.preventDefault()
 
+      // Get tab container ID
       var id = $(this).attr('href')
 
-      // Let the main tabs deal with the states
-      $('.app-c-tabs__item a[href="' + id + '"]').click()
+      // Activate current tab
+      GOVUK.tabs.activateCurrentTab(id)
```

Comment:
Could you check if `id` exists before doing this

Reply:
might be more bulletproof for each function that needs `id` to check that it has it before proceeding?